lib/RuntimeLib.lua

```diff
@@ -974,4 +974,32 @@ function TS.opcall(func, ...)
 	end
 end
 
+TS.TRY_RETURN = 1
+TS.TRY_BREAK = 2
+TS.TRY_CONTINUE = 3
+
+function TS.try(func, catch, finally)
+	local err, traceback
+	local success, exit_type, returns = xpcall(
```

Comment:
I'd prefer to use camel case in here where possible. `exit_type` -> `exitType`.

There's a few places that do not in the RuntimeLib, but that's because they used to be for automatic patterns. (i.e. `TS.array_reduce`)

Reply:
Oops, this is a habit from work 😅 I'll fix this.